cmd/kubectl-gadget/bcck8s/bcck8s.go

```diff
@@ -31,65 +31,13 @@ import (
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var profileCmd = &cobra.Command{
-	Use:   "profile",
-	Short: "Profile CPU usage by sampling stack traces",
-	RunE:  bccCmd("profile", "/usr/share/bcc/tools/profile"),
-}
-
-var (
-	stackFlag  bool
-	uniqueFlag bool
-
-	profileKernel bool
-	profileUser   bool
-)
-
-func init() {
-	commands := []*cobra.Command{
-		profileCmd,
-	}
-
-	// Add flags for all BCC gadgets
-	for _, command := range commands {
-		rootCmd.AddCommand(command)
-		utils.AddCommonFlags(command, &params)
-	}
-
-	profileCmd.PersistentFlags().BoolVarP(
-		&profileUser,
-		"user",
-		"U",
-		false,
-		"Show stacks from user space only (no kernel space stacks)",
-	)
-	profileCmd.PersistentFlags().BoolVarP(
-		&profileKernel,
-		"kernel",
-		"K",
-		false,
-		"Show stacks from kernel space only (no user space stacks)",
-	)
-}
-
-func bccCmd(subCommand, bccScript string) func(*cobra.Command, []string) error {
+func BccCmd(subCommand, bccScript string, params *utils.CommonFlags, gadgetSpecificFlag string) func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) error {
 		client, err := k8sutil.NewClientsetFromConfigFlags(utils.KubernetesConfigFlags)
 		if err != nil {
 			return utils.WrapInErrSetupK8sClient(err)
 		}
 
-		// tcptop only works on one pod at a time
```

Comment:
Why is this change part of this PR? Is this a rebase artifact or does it really belong here?

Reply:
This is dead code I deleted because I forgot to remove it from #555.
Anyway, once we wrote a golang control plane for `capabilities`, we could get rid of this file.

Reply:
> BTW, what do you think about working on such a golang control plane (only with the standard version) while the gadget naming is defined? It will give us the possibility to completely removed this file and make this PR simpler.

This is planned, I just want [this comment](https://github.com/iovisor/bcc/pull/3782#discussion_r829848289) to get an answer as I do not want to include code I do not 100% understand.

Reply:
Could the standard version of `profile` gadget be translated to golang regardless that comment? I would prefer to avoid blocking our process of moving to golang control plane for something that is not mandatory. The comment has been waiting for an answer for 26 days already :confused:.

Reply:
The standard can be translated right now.
But I prefer to add the golang control plane when we have the CO-RE version ready.

Reply:
Ok, let's go ahead with this PR as it is because it is blocking a lot of other PRs now. 

Beyond that, I would like to know, why do you want to wait for the CORE version if we already have the opportunity to finish the translation and remove that problematic `bcck8s` package? We already translated some gadgets to use the golang control plane without having a CORE version. Notice that the answer you are waiting for in BCC upstream could arrive after months (one month has already passed).

Reply:
> Beyond that, I would like to know, why do you want to wait for the CORE version if we already have the opportunity to finish the translation and remove that problematic bcck8s package?

I will not address this now, because it will take 2 days more to be sure the golang control plane is 100% safe and then it will delay this PR.

> We already translated some gadgets to use the golang control plane without having a CORE version.

In my case, I prefer to translate when we have everything and I think adding the golang control plane is a perfect pretext to write (or reuse) a CO-RE code.

> Notice that the answer you are waiting for in BCC upstream could arrive after months (one month has already passed).

Each thing in its own time.
I will ping them to get an answer (even though I do not like pinging people) and will see if I can begin to work to add this code.
I also have some PR which are opened since some time in bcc/iovisor and I do not find it problematic that they are waiting for reviews.
I cannot do antything to speed up the process and put pressure on people is clearly not a solution.